src/i18n/en/en.json

```diff
@@ -17,42 +17,42 @@
   },
   "welcome": {
     "complete_curriculum": "Begin or continue your studies with our complete curriculum covering all the religious sciences",
-    "mission_title": "Goal",
-    "mission": "Producing lordly scholars and righteous leaders, in order to revive the Religious Sciences",
+    "mission_title": "Purpose",
+    "mission": "To produce lordly scholars and righteous leaders, in order to revive the Religious Sciences",
     "online_title": "Library",
-    "online": "Featuring written articles and audio recordings",
+    "online": "Features written articles and audio and video recordings",
```

Comment:
I don't think we should mention video as it's not part of the Arabic, as Shaykh dictated the original wording. 

Video recordings *can* add a bit of stress and another concern to the teacher.

What do you think Sidi we circle back to this point once we (_if_ we) have video recordings available, and then update in Arabic and English together to mention video?

Reply:
My bad you're totally right Sidi